src/error_chain.rs

```diff
@@ -224,8 +224,8 @@ macro_rules! error_chain_processed {
 
         $(
             $(#[$meta_links])*
-            impl From<<$link_error_path as $crate::ChainedError>::ErrorKind> for $error_kind_name {
-                fn from(e: <$link_error_path as $crate::ChainedError>::ErrorKind) -> Self {
+            impl From<$link_kind_path> for $error_kind_name {
```

Comment:
Well, you keep ChainedError, but you also remove it main advantage, so what the point? Now you have to specify `ErrorKind` in `links`.